composer.json

```diff
@@ -17,14 +17,14 @@
         }
     ],
     "require": {
-        "psr/log": "^1.0",
+        "psr/log": "^1.0 || ^3.0",
```

Comment:
Why not allow version 2 as well?

Reply:
Good point.
I have updated it to support version 1,2 and 3